.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.11
+          python-version: 3.12
           cache: pip
           cache-dependency-path: netkan/setup.py
```

Comment:
```suggestion
```

If I'm understanding these changes, `cache-dependency-path` should probably be removed here as well since it no longer exists?

Reply:
Yes, I missed removing that!